modules/vba_documents/lib/vba_documents/sql_support.rb

```diff
@@ -1,236 +0,0 @@
-# frozen_string_literal: true
```

Comment:
These SQL queries were only used in the monthly report generation and have been replaced by `VBADocuments::MonthlyStatsGenerator`.